Decred Wallet.xcodeproj/project.pbxproj

```diff
@@ -714,6 +733,24 @@
 /* End PBXGroup section */
 
 /* Begin PBXNativeTarget section */
+		25F34E9023080A8100A999B2 /* Decred WalletTests */ = {
+			isa = PBXNativeTarget;
+			buildConfigurationList = 25F34E9823080A8200A999B2 /* Build configuration list for PBXNativeTarget "Decred WalletTests" */;
+			buildPhases = (
+				25F34E8D23080A8100A999B2 /* Sources */,
+				25F34E8E23080A8100A999B2 /* Frameworks */,
+				25F34E8F23080A8100A999B2 /* Resources */,
+			);
+			buildRules = (
+			);
+			dependencies = (
+				25F34E9723080A8200A999B2 /* PBXTargetDependency */,
+			);
+			name = "Decred WalletTests";
+			productName = "Decred WalletTests";
+			productReference = 25F34E9123080A8100A999B2 /* Decred WalletTests.xctest */;
+			productType = "com.apple.product-type.bundle.unit-test";
```

Comment:
Don't know why this change is necessary. Seems you attempted to add unit-testing. This would be related to https://github.com/raedahgroup/dcrios/pull/516/files#r324634354. See if you can undo this and other unrelated changes to this file. One possible approach is to open master branch's version of `Decred Wallet.xcodeproj/project.pbxproj` as a text file, copy over the text contents there over here and re-link `FloatingLabelTextInput.swift` and unlink `PinInputView.swift`.

Reply:
resolved in 07f7e91